### PR TITLE
Comment and cleanup of heat balance arrays

### PR DESCRIPTION
**Pull request by xuanluo113**
Pull request overview
---------------------
This branch includes a few naming convention cleanups and a few local variable refactoring in calculating interior solar distribution.

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [x] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [x] Perform a Code Review on GitHub
 - [x] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [x] CI status: all green or justified
 - [x] Check that performance is not impacted (CI Linux results include performance check)
 - [x] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by xuanluo113** [comment]
@mjwitte This branch is ready for review. It's no more than a few naming convention cleanups and a few local variable refactoring in calculating interior solar distribution. Would you help take a look? Thanks!

File: src/EnergyPlus/DataHeatBalance.hh [hunk 1]
```diff
@@ -578,7 +578,14 @@ namespace DataHeatBalance {
         int ZoneEqNum;                            // Controlled zone equip config number
         int PlenumCondNum;                        // Supply or return plenum conditions number, 0 if this is not a plenum zone
         int TempControlledZoneIndex;              // this is the index number for TempControlledZone structure for lookup
-        //            Pointers to Surface Data Structure
+        // Pointers to Surface Data Structure
+        // AllSurfF                                                                   AllSurfL
```
Comment (mjwitte) [inline src/EnergyPlus/DataHeatBalance.hh]: This is very helpful, but I would pay close attention to the alignment and possibly add `|` characters to draw vertical lines to show the groupings. 
Also, some of these will equal each other, e.g. `HTSurfaceFirst = OpaqOrIntMassSurfaceFirst`. 
And comments explaining how these are set if a particular class of surfaces are not present in the zone. e.g. if there are no domes, what are `TDDDomeFirst and TDDDomeLast` set to?

**Review by mjwitte — COMMENTED** [review]
@xuanluo0113 This looks good. Just a minor comment below.
~Also, I don't see any CI results posted on the PR, so I'll check those after the next push.~

File: src/EnergyPlus/DataHeatBalance.hh (hunk 1 above)
Comment (xuanluo113) [reply src/EnergyPlus/DataHeatBalance.hh]: @mjwitte Thanks for the comments! I've committed the changes to make the comments clearer. For TDDDomes, same as Windows, if there are no domes, `TDDDomeFirst` is set as 0 and `TDDDomeLast` as -1.

**Review by mjwitte — APPROVED** [review]
@xuanluo113 Thanks. Much better. Will wait for CI green lights, then merge.

**Pull request merged**